app/public/static/js/components/keypad.js

```diff
@@ -38,7 +38,7 @@ AFRAME.registerComponent("keypad", {
     }
 
     el.addEventListener("superkeyboardinput", function (event) {
-      if (event.detail.value === password) {
+      if (event.detail.value.toLowerCase() === password.toLowerCase()) {
```

Comment:
Todo for later: allow admin to specific whether they want case sensitive or not -- case sensitivity might be important for some custom admin created puzzles!